superset/assets/src/explore/components/controls/AnnotationLayer.jsx

```diff
@@ -464,6 +466,15 @@ export default class AnnotationLayer extends React.PureComponent {
                   }
                 }}
               />
+              <TextControl
+                hovered
+                name="annotation-override-param"
+                label="Override params"
+                description={'Override the params in the chart'}
```

Comment:
Looks like they use back ticks wrapped in curly braces elsewhere in this file. I'd guess it's either for localization or so they can use quotes in the their strings without goofy escapes.

Change this to ``description={`Override the params in the chart`}`` for consistency

Reply:
I used single quotes so that this file passes eslint. It looks like backquotes are used for multi-line strings